salt/pillar/consul_pillar.py

```diff
@@ -131,6 +131,9 @@ def fetch_tree(client, path):
 
     log.debug('Fetched items: %r', format(items))
 
+    if items is None:
+        return ret
+        
```

Comment:
There is some trailing whitespace here. Could you please remove it before we merge this? Thanks.